### PR TITLE
ER-1122: Lead on news

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ER-1122

Depends on changes in https://github.com/eReolen/ereolengo/pull/43

https-jira-itkdev-dk-browse-ER-1122-eReolen-GO-

### CODE REVIEW

File: sites/all/themes/wille/assets/sass/component/_article_breol_news.scss [hunk 1]
```diff
@@ -29,6 +29,9 @@
     margin-top: 20px;
     padding: 0 100px;
 
+    font-family: $font-regular;
```
Comment (yepzdk) [inline sites/all/themes/wille/assets/sass/component/_article_breol_news.scss:32]: Won't this override the selected styles from the editor?

**Review by yepzdk — COMMENTED** [review]
Just a single question regarding css.

File: sites/all/themes/wille/assets/sass/component/_article_breol_news.scss (hunk 1 above)
Comment (rimi-itk) [reply sites/all/themes/wille/assets/sass/component/_article_breol_news.scss:32]: Good point, but I think https://github.com/eReolen/base/blob/feature/ER-1122-lead-on-news/sites/all/themes/wille/assets/sass/base/_base.scss#L55-L57 saves the day.

Comment (yepzdk) [reply sites/all/themes/wille/assets/sass/component/_article_breol_news.scss:32]: Alright then. Good work.

**Review by yepzdk — APPROVED** [review]

**Review by cableman — APPROVED** [review]
If https://github.com/eReolen/base/pull/166 is approved

**Review by cableman — APPROVED** [review]

**Pull request merged**